packages/manager/modules/overthebox/src/overthebox/remote/overTheBox-remote.controller.js

```diff
@@ -171,8 +173,7 @@ export default class OverTheBoxRemoteCtrl {
    * @param {Date} when Date to compare
    * @return {Boolean}
    */
-  // eslint-disable-next-line class-methods-use-this
-  isInFuture(when) {
+  static isInFuture(when) {
```

Comment:
Regarding the rest of the diff, it looks like a `refactor` because class methods
has been just turned into static ones.